src/main/java/nl/avans/drivio/model/FuelCar.java

```diff
@@ -1,15 +1,16 @@
 package nl.avans.drivio.model;
 

```

Comment:
Dit hele bestand is anders in mijn code, ik zou deze niet gebruiken en mijn code hiervoor gebruiken

Reply:
Fixed met merge conflicts in commit https://github.com/leonvsc/Drivio/pull/26/commits/d6c224c192476645187e85243bfa37e53c615ef4